service/resources/welcome.py

```diff
@@ -14,5 +14,5 @@ def on_get(self, _req, resp):
         return Welcome message
         """
         msg = {'message': 'Welcome'}
-        resp.body = json.dumps(jsend.success(msg))
+        resp.text = json.dumps(jsend.success(msg))
```

Comment:
Just curious, what's the difference between resp.body and resp.text?

Reply:
I believe they are the same - however, resp.body is being deprecated